target/private_include/ll_cam.h

```diff
@@ -64,6 +64,12 @@
 
 #define LCD_CAM_DMA_NODE_BUFFER_MAX_SIZE  (4092)
 
+#if CONFIG_CAM_TASK_STACK_SIZE
+#define CAM_TASK_STACK             CONFIG_CAM_TASK_STACK_SIZE
+#else
+#define CAM_TASK_STACK             (2*1024)
+#endif
```

Comment:
It's a little bit weird that the task size related code ends up in the "ll" file. Since CAM_TASK_STACK is only used in driver/cam_hal.c, could this code block be moved there?

Reply:
Moved the relevant code to the driver/cam_hal.c, PTAL, thanks.